yb-voyager/src/query/queryissue/issues_ddl_test.go

```diff
@@ -128,12 +148,33 @@ func testUnloggedTableIssue(t *testing.T) {
 
 }
 
+func testAlterTableAddPKOnPartitionIssue(t *testing.T) {
+	ctx := context.Background()
+	conn, err := getConn()
+	assert.NoError(t, err)
+
+	defer conn.Close(context.Background())
+	_, err = conn.Exec(ctx, `
+	CREATE TABLE orders2 (
+	order_id bigint NOT NULL,
+	order_date timestamp
+	) PARTITION BY RANGE (order_date);
+	ALTER TABLE orders2 ADD PRIMARY KEY (order_id,order_date)`)
+
+	assertErrorCorrectlyThrownForIssueForYBVersion(t, err, "changing primary key of a partitioned table is not yet implemented", alterTableAddPKOnPartitionIssue, testYbVersion)
+}
+
 func TestDDLIssuesInYBVersion(t *testing.T) {
+	var err error
 	ybVersion := os.Getenv("YB_VERSION")
 	if ybVersion == "" {
 		panic("YB_VERSION env variable is not set. Set YB_VERSIONS=2024.1.3.0-b105 for example")
 	}
 
+	ybVersionWithoutBuild := strings.Split(ybVersion, "-")[0]
```

Comment:
Q: so our `--target-db-version` can only accept the version without build number only?
Just asking this if we are mentioning this in flag's help msg ?

Reply:
So, right now it does accept build number as well, but the problem is that `2024.2.0.0-b145` is considered to be > `2024.2.0.0` (which is defined in the MinVersionsFixedIn). 

I'm planning to raise another PR preventing users from passing the build number in the input to prevent these kinds of issues.